src/Message/ButtonReplyMessage.php

```diff
@@ -8,29 +8,21 @@ class ButtonReplyMessage extends Message
 {
     protected string $type = 'button';
 
-    private ?string $header;
-
     private string $body;
 
     private ?string $footer;
 
     private ButtonAction $action;
 
-    public function __construct(string $to, string $body, ButtonAction $action, ?string $header = null, ?string $footer = null, ?string $reply_to = null)
+    public function __construct(string $to, string $body, ButtonAction $action, ?string $footer = null, ?string $reply_to = null)
     {
         $this->body = $body;
         $this->action = $action;
-        $this->header = $header;
         $this->footer = $footer;
 
         parent::__construct($to, $reply_to);
     }
 
-    public function header(): ?string
```

Comment:
header is a public method, if you remove a public method you are breaking all projects that update the version of this package.

Reply:
Same as reply given before